TensorFlow2/Segmentation/MaskRCNN/mask_rcnn_sm.py

```diff
@@ -0,0 +1,94 @@
+# Copyright (c) 2020, NVIDIA CORPORATION. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" Scripts that simplifies running training benchmark """
+
+import argparse
+import os
+import shutil
+import subprocess
+from argparse import ArgumentError
+
+
+def csv_int(vstr, sep=','):
+    values = []
+    for v0 in vstr.split(sep):
+        try:
+            v = int(v0)
+            values.append(v)
+        except ValueError as err:
+            raise ArgumentError('Invalid value %s, values must be a number' % v)
+    return values
+
```

Comment:
Can you please 1 or 2 line of description to say what was the problem and why this parsing was required. Makes it more explicit for other users and team members.

Reply:
Done. e5dfc63c93165bf23a058c3fc5b34b7a2928bd4a